web_widget_slickroom/README.rst

```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===================================================
+Slick Carousel Widget with DarkroomJS Image Editing
+===================================================
+
+This module extends the `Slick`_ carousel widget provided by
+`web_widget_slick` to include the `DarkroomJS`_ image editing features provided
+by `web_widget_darkroom`.
+
+.. _Slick: http://kenwheeler.github.io/slick
+.. _DarkroomJS: https://github.com/MattKetmo/darkroomjs
+
+Usage
+=====
+
+To create a Slick carousel widget with DarkroomJS support, follow the
+usage instructions in the `web_widget_slick` documentation, but replace
+"one2many_slick_images" with "slickroom" in the field definition, as shown
+here::
+
+    <field name="image_ids" widget="slickroom" options="{}"/>
+
+To edit an image in a carousel, simply click on it to open a DarkroomJS modal,
+edit the image as described in the `web_widget_darkroom` documentation, and
+click save to save the changes and update the carousel.
+
+Example Module
+--------------
+
+The `web_widget_slick_example` module (see `web_widget_slick` documentation for
```

Comment:
Go ahead and just make the example module dependent on this module, then add another example of how to use the slickroom widget. There should then be two examples within the page - one for Slick and one SlickRoom